### ISSUE TITLE
Time delay estimation options

### ISSUE THREAD

**Issue by drphilmarshall** [DarkEnergyScienceCollaboration/SLTimer#22]
I think we want the `estimate_time_delays` to have some more options. For example, we need to separate the intrinsic variability model from the microlensing model. How about:

``` python
timer.estimate_time_delays(method=`pycs`, agn='spline', microlensing=`polynomial`)
```

and then further kwargs for the key parameters of these methods?

**Comment by milanwilliams** [comment]
This is now implemented and will be merged with #26

**Comment by drphilmarshall** [comment]
Great!
